cmd/krew/cmd/internal/setup_check.go

```diff
@@ -58,7 +59,11 @@ func IsBinDirInPATH(paths environment.Paths) bool {
 
 	binPath := paths.BinPath()
 	for _, dirInPATH := range filepath.SplitList(os.Getenv("PATH")) {
-		if dirInPATH == binPath {
+		normalizedDirInPATH, err := filepath.Abs(dirInPATH)
```

Comment:
```suggestion
		dirInPATH, err := filepath.Abs(dirInPATH)
```
It's fine to reuse the old name here.

Reply:
Now I reuse old name dirInPath but i am not sure that it is a good idea. Indeed if `filepath.Abs(dirInPATH)` function call fail we will have dirInPath set with wrong value for logging message line 64 ?

cf code here :

https://github.com/kubernetes-sigs/krew/pull/665/files#diff-8e3a2729df3d5b4798b7b59ea867eef83cff5f821442bd739451e5f059155cd2R61-R66

Reply:
Oops. I thought I had already responded that you are right here, but apparently that reply got lost 😅 

I don't think it is critical, but it would be cleaner to change that.